definitions/divorce/json/CaseTypeTab/CaseTypeTab.json

```diff
@@ -3464,50 +3464,204 @@
     "TabFieldDisplayOrder": 7
   },
   {
-    "LiveFrom": "12/12/2019",
+    "LiveFrom": "01/03/2021",
```

Comment:
I don't think liveFrom works anymore does it?

Reply:
I'm not sure.

Reply:
https://tools.hmcts.net/confluence/display/RCCD/CCD+Definition+Glossary+for+Setting+up+a+Service+in+CCD#CCDDefinitionGlossaryforSettingupaServiceinCCD-ccdDefTabCaseTypeTab

sorry my mistake - 'liveTo' doesn't work anymore - live from seemingly seems to still work